src/__tests__/__snapshots__/get-by-errors.js.snap

```diff
@@ -0,0 +1,205 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`getAllByAltText query will throw expected error message when config.showLogOnFail is false 1`] = `"Unable to find an element with the alt text: TEST QUERY"`;
+
+exports[`getAllByAltText query will throw expected error message when config.showLogOnFail is true 1`] = `
+"Unable to find an element with the alt text: TEST QUERY
+
+<body>
+  <div>
+    Hello
+  </div>
+</body>"
+`;
```

Comment:
Honestly, I think just testing two of the queries is sufficient. The rest is just noise IMO.

Reply:
Agreed. Just updated.